notify-send.sh

```diff
@@ -125,7 +125,7 @@ notify() {
     fi
 
     if [[ -n "$FORCE_EXPIRE" ]] ; then
-        SLEEP_TIME="$( printf %f "${EXPIRE_TIME}e-3" )"
+        SLEEP_TIME="$( LC_NUMERIC='en_US.UTF-8' printf %f "${EXPIRE_TIME}e-3" )"
```

Comment:
This likely causes issues on systems which don't have `en_US.UTF-8` installed. `LC_NUMERIC='C'` should do the job.

Reply:
Thank you! I just fixed that